lib/font/afm.js

```
@@ -0,0 +1,227 @@
+        section = match[1];
+        continue;
+      
+      } else if (match = line.match(/^End(\w+)/)) {
+        section = '';
+        continue;
+      }
+      
+      switch (section) {
+        case 'FontMetrics':
+          match = line.match(/(^\w+)\s+(.*)/);
+          var key = match[1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)